src/include/session.h

```diff
@@ -40,7 +40,7 @@ struct __wt_hazard {
 typedef TAILQ_HEAD(__wt_cursor_list, __wt_cursor)	WT_CURSOR_LIST;
 
 /* Number of cursors cached to trigger cursor sweep. */
-#define	WT_SESSION_CURSOR_SWEEP_COUNTDOWN	20
+#define	WT_SESSION_CURSOR_SWEEP_COUNTDOWN	40
```

Comment:
Why is it increased to 40?

Reply:
Currently, there are 2 places to trigger the cursor cache sweep(`__wt_session_cursor_cache_sweep`): `__curfile_close` and `__session_reset`. Before the change, the countdown variable (`session->cursor_sweep_countdown`) is only used from `__curfile_close` and not from `__session_reset`. That's why `__wt_epoch` was called every time we reset the session.  
Now in this change, because we use the same countdown variable to trigger less cursor cache sweep from `__session_reset` just like `__curfile_close`, I doubled the initial variable 20 to 40.